app/policies/comment_policy.rb

```diff
@@ -8,7 +8,7 @@ def new?
   def update?
     return true if user.admin
 
-    record.user == user
+    owner? && record.redacted? == false
```

Comment:
Not sure if it's just me, but 

`owner? && !record.redacted?`

Feels easier to read. Might be because the operator precedence of && and == is so similar so I had to double check whether it would evaluate as (owner? && record.redacted?) == false or  owner? && (record.redacted? == false)

Reply:
Ok makes sense, I've changed